AllReadyApp/Web-App/AllReady/Views/Manage/Index.cshtml

```diff
@@ -65,7 +82,8 @@
     @using (Html.BeginForm())
     {
         @Html.AntiForgeryToken()
-
+        <input asp-for="IsProfileComplete" type="hidden"/>
+        <input asp-for="ProfileCompletenessWarnings" type="hidden" />

```

Comment:
These hidden fields aren't really needed. This information isn't used in the Manage/Index post action and the value of ProfileCompletenessWarnings ends up getting garbled anyway: `System.Linq.Enumerable+WhereSelectListIterator`2[System.ComponentModel.DataAnnotations.ValidationResult,System.String]`. Even if you did need it in the post action, the information is easily re-computed.
Are these hidden fields needed? This isn't any information that the